runtime/plugin/net.lua

```diff
@@ -0,0 +1,43 @@
+vim.g.loaded_remote_file_loader = true
+
+-- Only load if curl is on the path
+if vim.fn.executable('curl') ~= 1 then
+  vim.api.nvim_echo({
+    { 'Warning: `curl` not found; remote file loading disabled.', 'WarningMsg' },
+  }, true, {})
+  return
+end
+
+-- Autocmd group for remote URLs
+local aug = vim.api.nvim_create_augroup('NetRemoteFile', { clear = true })
```

Comment:
```suggestion
local aug = vim.api.nvim_create_augroup('NetRemoteFile', { clear = true })
```
`clear = true` is the default, so you can just put `group = vim.api.nvim_create_augroup('NetRemoteFile', {})` in the `create_autocmd` call. (You only need to bind it if you want to reuse it multiple times.)